src/meta/processors/partsMan/CreateSpaceProcessor.cpp

```diff
@@ -71,16 +72,30 @@ void CreateSpaceProcessor::process(const cpp2::CreateSpaceReq& req) {
     if (replicaFactor == 0) {
         replicaFactor = FLAGS_default_replica_factor;
         if (replicaFactor <= 0) {
-            LOG(ERROR) << "Create Space Failed : replicaFactor is illegal!";
-              resp_.set_code(cpp2::ErrorCode::E_INVALID_PARM);
-              onFinished();
-              return;
+            LOG(ERROR) << "Create Space Failed : replicaFactor is illegal: " << replicaFactor;
+            resp_.set_code(cpp2::ErrorCode::E_INVALID_PARM);
+            onFinished();
+            return;
         }
         // Set the default value back to the struct, which will be written to storage
         properties.set_replica_factor(replicaFactor);
     }
     if (vidSize <= 0 && vidSize > std::numeric_limits<int32_t>::max()) {
```

Comment:
graphd limit it less than std::numeric_limits<int16_t>::max(). here need to unify